pkg/image/trigger/util/resolve.go

```diff
@@ -0,0 +1,31 @@
+package util
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	imageapi "github.com/openshift/origin/pkg/image/apis/image"
+	triggerapi "github.com/openshift/origin/pkg/image/apis/image/v1/trigger"
+	imageclient "github.com/openshift/origin/pkg/image/generated/internalclientset/typed/image/internalversion"
+)
+
+// ResolveLatestTriggerImagePullSpec resolves the latest image pull spec for
+// given object reference.
+func ResolveLatestTriggerImagePullSpec(c imageclient.ImageStreamTagsGetter, namespace string, ref triggerapi.ObjectReference) (*imageapi.DockerImageReference, error) {
```

Comment:
Put this in `pkg/image/trigger/resolve/resolve.go` and change the name `LatestTriggerImagePullSpec`.  I hate `util` :)